### PR TITLE
#73 - Add get_covered_text method to Annotations

### PR DESCRIPTION
**Pull request by reckart**
- Make annotation.sofa an actual reference to the sofa instead of just storing its XMI ID
- During serialization, fetch the XMI ID and write it out

### CODE REVIEW

File: cassis/typesystem.py [hunk 1]
```diff
@@ -125,6 +125,18 @@ class FeatureStructure:
     def __eq__(self, other):
         return self.__slots__ == other.__slots__
 
+    def get_covered_text(self):
```
Comment (jcklie) [inline cassis/typesystem.py]: The return type is missing

File: tests/fixtures.py [hunk 2]
```diff
@@ -210,28 +210,43 @@ def inception_typesystem_xml(inception_typesystem_path):
 def tokens(small_typesystem_xml):
     typesystem = load_typesystem(small_typesystem_xml)
 
+    cas = Cas(typesystem)
```
Comment (jcklie) [inline tests/fixtures.py]: Why do you create a cas here and then dont use it?

Comment (reckart) [reply tests/fixtures.py]: The tokens are added to it and as part of this, the sofa is set on the tokens.

File: cassis/typesystem.py (hunk 1 above)
Comment (reckart) [reply cassis/typesystem.py]: Added.

**Pull request merged**